fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/WindowTransformer.java

```diff
@@ -117,7 +118,12 @@ public static WindowOperator standardize(AnalyticExpr analyticExpr) {
                     throw new SemanticException("Convert type error in offset fn(default value); old_type="
                             + callExpr.getChild(2).getType() + " new_type=" + callExpr.getChild(0).getType());
                 }
-                if (callExpr.getChild(2) instanceof CastExpr) {
+
+                // Allow bitmap_empty/hll_empty as third paramter of LEAD/LAG.
+                if (!((callExpr.getFnName().getFunction().equalsIgnoreCase(FunctionSet.LEAD) || 
```

Comment:
I think needn't cast for bitmap and hll.

Reply:
Removed.